src/gui/advancedsettings.h

```diff
@@ -57,9 +57,10 @@ private slots:
 private:
     void loadAdvancedSettings();
     template <typename T> void addRow(int row, const QString &text, T *widget);
+    QString makeLink(const QString &url, const QString &linkLabel);
```

Comment:
It's nothing to do with the AdvancedSettings class so you should move it out.
Also please correct commit message.

Reply:
>you should move it out

As in above/below the class or remove it from the header file completely?

Reply:
In anonymous namespace in .cpp.

Reply:
Done.